tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -91,6 +94,70 @@ TfLiteStatus AllocateVariables(
   return kTfLiteOk;
 }
 
+
+// Helper function to check flatbuffer metadata correctness. This function is
+// not called by default. Hence it's not linked in to the final binary code.
+TfLiteStatus CheckOfflinePlannedOffsets(const Model* model,
+                                        ErrorReporter* error_reporter) {
+  // Suppress compile warning for unused function
+  (void)CheckOfflinePlannedOffsets;
```

Comment:
`tensorflow/lite/micro/micro_allocator.cc:102:14: error: function 'CheckOfflinePlannedOffsets' is not needed and will not be emitted [-Werror,-Wunneeded-internal-declaration]
TfLiteStatus CheckOfflinePlannedOffsets(const Model* model,`

Reply:
From what I could see when I tested it, that error showed up when using clang.
I solved it like this, which might not be optimal. What do you think about it?
https://github.com/tensorflow/tensorflow/pull/38121/commits/bacc5e5927360ad119cdb9967d53df4c95c53fd4#diff-6dace628e5372fb3c3fbb76e0fd9d37aR97